src/app/home-page/components/finish-modal/finish-modal.component.html

```diff
@@ -0,0 +1,7 @@
+<div [ngClass]="{ stealth: visibility }">
+  <div class="finish-frame">
+    <span class="finish-text">
+      Great work. You just finish 1 set of pomodoro. Take a short break!
+    </span>
```

Comment:
What about an idea to make this modal appear after breaks too, with appropriate notifications?

Reply:
That's a good point. As I understand it, the modal appears when the Pomodoro session finishes. In that case, we need to create a new modal. Thank you for your comment. I found a bug in my implementation of this modal - it appears when the break ends.